modules/identity/src/Volo.Abp.Identity.MongoDB/Volo/Abp/Identity/MongoDB/MongoOrganizationUnitRepository.cs

```diff
@@ -155,7 +161,18 @@ public virtual async Task RemoveAllMembersAsync(OrganizationUnit organizationUni
             }
         }
 
-        protected virtual IMongoQueryable<IdentityUser> CreateGetMembersFilteredQuery(OrganizationUnit organizationUnit, string filter = null)
+        public virtual async Task<long> GetLongCountAsync(Guid? parentId, string filter = null,
```

Comment:
Same unused `parentId` parameter.

Reply:
I have already solved.

1. I think it is necessary for GetListAsync to filter through parentId when the OrgnizationUnit is more complex.

**Example:**
- The organization structure tree is lazily loaded.
- When there are many suborganizational units.

Reply:
I can't see updated code on View Change set. Can you update your dev branch?

Reply:
lasted commit id 79f1369(resolved conflicts).